helm/dagster/values.yaml

```diff
@@ -583,6 +587,10 @@ postgresql:
 # global.postgresqlSecretName is managed outside of this helm chart.
 generatePostgresqlPasswordSecret: true
 
+# Whether to generate a secret resource for Celery config. Useful if global.celeryConfigSecretName
```

Comment:
Kind of confusing- useful to set this to false

Maybe

> If true, the helm chart will create a secret for you with the Celery broker and backend connection urls, including an optional Redis or RabbitMQ passwords. Set this to false if you want to manage a secret with the Celery broker/backend connection strings yourself. This secret should have the name specified in global.celeryConfigSecretName

Plus include the env var names as above